src/System.Private.CoreLib/shared/System/Runtime/Serialization/SerializationInfo.cs

```diff
@@ -19,19 +19,25 @@ public sealed class SerializationInfo
         private Type[] _types;
         private int _count;
         private Dictionary<string, int> _nameToIndex;
-
         private IFormatterConverter _converter;
         private string _rootTypeName;
         private string _rootTypeAssemblyName;
         private Type _rootType;
 
         [CLSCompliant(false)]
         public SerializationInfo(Type type, IFormatterConverter converter)
+            : this(type, converter, false)
+        {
+        }
+
+        [CLSCompliant(false)]
+        public SerializationInfo(Type type, IFormatterConverter converter, bool requireSameTokenInPartialTrust)
```

Comment:
This constructor should be remove too. It is not part of public surface.

Reply:
my bad, I should've noticed

Reply:
Ignore this. It is part of public surface... I have have not noticed that it just moved around.